imports/api/groceries.js

```diff
@@ -50,14 +52,32 @@ Meteor.methods({
 			$set: { archived }
 		});
 	},
-	'groceries.setChecked'(groceryId, setChecked) {
+	'groceries.setChecked'(groceryId, setChecked, addToInventory = true) {
 		check(groceryId, String);
 		check(setChecked, Boolean);
 		authCheck(Groceries, this.userId, groceryId);
 
-		return Groceries.update(groceryId, {
+		Groceries.update(groceryId, {
 			$set: { checked: setChecked }
 		});
+
+		const grocery = Groceries.findOne(groceryId);
+
+		if (addToInventory) {
+			Inventories.insert({
```

Comment:
I'm thinking of pulling this out into a function in `inventories.js` and then exporting it so that this file can just use the exported function